package.json

```diff
@@ -37,7 +37,7 @@
     "react-dom": "^18.2.0",
     "use-debounce": "^10.0.0",
     "viem": "^1.10.8",
-    "wagmi": "1.4.12"
+    "wagmi": "^1.4.12"
```

Comment:
Thoughts on pinning across our repos? Good points raised here: https://docs.renovatebot.com/dependency-pinning/

Reply:
💯 was gonna discuss the same.